phpcs.xml.dist

```diff
@@ -6,8 +6,10 @@
 
     <config name="installed_paths" value="../../slevomat/coding-standard"/>
 
-    <!--The complete PSR-2 ruleset-->
-    <rule ref="PSR2"/>
+    <!-- The complete PSR-12 ruleset (excluding a rule that requires PHP >= 7.1 -->
+    <rule ref="PSR12">
+        <exclude name="PSR12.Properties.ConstantVisibility"/>
```

Comment:
I should have read the comment above :/.  I just wasted a few minutes trying to understand what this was about.